clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10250,6 +10255,9 @@ void OffloadPackager::ConstructJob(Compilation &C, const JobAction &JA,
         C.getArgsForToolChain(TC, OffloadAction->getOffloadingArch(),
                               OffloadAction->getOffloadingDeviceKind());
     StringRef File = C.getArgs().MakeArgString(TC->getInputFilename(Input));
+    if (Input.getType() == types::TY_Tempfilelist)
```

Comment:
Can you please add a comment about why we are adding this prefix? Thanks

Reply:
done, thanks!